admin/load.php

```diff
@@ -757,58 +791,74 @@ function perflab_plugin_admin_notices() {
 		<?php
 	}
 
-	$active_modules_with_plugins = perflab_get_active_modules_with_standalone_plugins();
+	$available_module_names = perflab_get_modules_compatible_with_standalone_plugin();
+	if ( empty( $available_module_names ) ) {
+		return;
+	}
```

Comment:
See my other comment: Showing the names is a UI specific detail that shouldn't be baked into the API functions. If we change the function to return the full module data, we can simply change this to e.g.:
```suggestion
	$available_modules_data = perflab_get_active_module_data_with_inactive_standalone_plugins();
	if ( empty( $available_modules_data ) ) {
		return;
	}

	$available_module_names = wp_list_pluck( $available_modules_data, 'name' );
```